### DISCUSSION TITLE
'definePage' is not defined. eslint(no-undef)

### DISCUSSION THREAD

**Discussion by qiuqimin** [posva/unplugin-vue-router#511] [Questions and Help] [answered]
Already use unplugin-auto-import according to https://github.com/unplugin/unplugin-auto-import?tab=readme-ov-file#eslint. Still got eslint error 'definePage' is not defined. And, the code runs normally.

Can I avoid manually importing 'definePage'?

repo: https://github.com/qiuqimin/unplugin-vue-router-auto-imports-eslint
example file: src/views/about.vue

**Comment by posva** [comment, marked as answer]
You need to disable in [eslint config](https://eslint.org/docs/latest/use/configure/language-options#using-configuration-files) or you can safely disable the rule if you use TypeScript.

I added this to docs: 2126ea0

**Comment by qiuqimin** [comment]
@posva Once I configure VueRouterAutoImports, can 'definePage' be automatically imported like 'useRoute' or 'useRouter' without having to manually handle ESLint errors?

**Reply by posva** [reply]
It’s not there. It’s a global macro
